pages/events.js

```diff
@@ -0,0 +1,271 @@
+  },
+];
+
+const hostEventItems = [
+  {
+    icon: <BullhornIcon />,
+    label: 'Raise Awareness',
+  },
+  {
+    icon: <DiversityIcon />,
+    label: 'Add Diversity to Conventions',
+  },
+  {
+    icon: <UserIcon />,
+    label: 'Get Our Voices out There',
+  },
+];
+
+/* commented out until api endpoints */
+// const carouselItems = [
+//   {
+//     label: 'OpCode North Carolina: Durham',
+//     date: '10/06/2018',
+//   },
+//   {
+//     label: 'Hacktober',
+//     date: '10/01/2018',
+//   },
+//   {
+//     label: 'OpCode Boston',
+//   },
+//   {
+//     label: 'OpCode North Carolina: Durham',
+//     date: '10/06/2018',
+//   },
+//   {
+//     label: 'Hacktober',
+//     date: '10/01/2018',
+//   },
+//   {
+//     label: 'OpCode Boston',
+//   },
+//   {
+//     label: 'OpCode North Carolina: Durham',
+//     date: '10/06/2018',
+//   },
+// ];
+
+export default () => {
+  const pageTitle = 'Events';
+
+  return (
+    <>
+      <Head title={pageTitle} />
+
+      <HeroBanner title={pageTitle} backgroundImageSource={`${s3}redesign/heroBanners/events.jpg`}>
+        <>
+          <p>
+            Need more Operation Code in your life? From conventions to tech meetups, we are where
+            you are! There are 4,000+ members across more than 20 states!
+          </p>
+
+          <LinkButton href="#">See Locations</LinkButton>
+        </>
+      </HeroBanner>
+
+      <Content
+        title="Upcoming Events:"
+        theme="gray"
+        columns={[
+          <p className={styles.eventsLeadText}>
+            Be sure to check back often or join your local meetup for a more up to date list of
+            meetups or engagements
+          </p>,
+        ]}
+      />
+
+      {/* commented out until api endpoints */}
+      {/* <div className={styles.eventsCarousel}>
+        <Carousel
+          slideWidth="325px"
+          renderCenterLeftControls={({ previousSlide }) => (
+            <button type="button" className={styles.carouselControlLeft} onClick={previousSlide}>
+              <LeftAngleIcon />
+            </button>
+          )}
+          renderCenterRightControls={({ nextSlide }) => (
+            <button type="button" className={styles.carouselControlRight} onClick={nextSlide}>
+              <RightAngleIcon />
+            </button>
+          )}
+          renderBottomCenterControls={null}
+        >
+          {carouselItems.map(item => (
+            <div className={styles.eventsCarouselItem} key={item.label}>
+              {item.label}
+              {item.date && <div className={styles.eventsCarouselItemDate}>{item.date}</div>}
+            </div>
+          ))}
+        </Carousel>
+      </div> */}
+
+      <HeroBanner
+        backgroundImageSource={`${s3}redesign/heroBanners/events.jpg`}
+        className={styles.meetupCardsBanner}
+      >
+        <div className={styles.meetupCardsRow}>
+          <div className={styles.col}>
+            <FlatCard className={styles.meetupCard}>
```

Comment:
While adjusting the FlatCard to allow for no image to be passed like we discussed on Slack, I wonder if you could also define an opinionated way for a LinkButton or Button to show up along the bottom bar 🤔 

These exact cards also appear on the code schools page for MooCs, so it'd be used in many places (although those FlatCards have images AND the bottom bar button). I've got ideas on how to give FlatCard that prop, but am interested in seeing how you'd solve it.